content/blog/2022-05-06-end-to-end-computer-vision-api-part-3-remote-exp-ci-cd.md

```diff
@@ -0,0 +1,292 @@
+---
+title:
+  'End-to-End Computer Vision API, Part 3: Remote Experiments & CI/CD For
+  Machine Learning'
+date: 2022-05-06
+description: >
+  This is the third and last part of this series of posts on how to build an
+  end-to-end Computer Vision solution according to the best available MLOps
+  practices. 
+
+  In part 1, we talked about exploratory work in Jupyter Notebooks; versioning
+  data in remote storage with DVC; and refactoring the code from Jupyter
+  Notebooks into DVC pipeline stages.
+
+  Part 2 talked about the process of managing experiments with DVC pipelines,
+  DVCLive and Iterative Studio.
+
+descriptionLong: |
+
+  This is the third and last part of this series of posts on how to build an end-to-end Computer Vision solution according to the best available MLOps practices. 
```

Comment:
please format it (under 80 symbols) - unfortunately this is the only place if we want to have some complicated MD that we can't automate the formatting for